iquip/apps/scheduler.py

```diff
@@ -0,0 +1,333 @@
+        if info:
+            self.nameLabel.setText(info.name)
+            for key, value in info.arginfo.items():
+                if key != "priority":
+                    self.argsLayout.addWidget(QLabel(f"{key}: {value}", self))
+        else:
+            self.nameLabel.setText("None")
+
+class ExperimentView(QWidget):
+    """Widget for displaying the information the experiment.
+    
+    Attributes:
+        layout: The list of ExperimentView widget.
+        argsLayout: The HBoxLayout for displaying the experiment information besides its name.
+        nameLabel: The QLabel instance for displaying the experiment name.
+        editButton: The button to call frame for edition of the experiment.
+    """
+
+    def __init__(self, info: ExperimentInfo, parent: Optional[QWidget] = None):
+        """Extended.
+        
+        Args:
+            info: The information of the experiment.
+        """
+        super().__init__(parent=parent)
+        self.experimentInfo = info
+        # widgets
+        self.nameLabel = QLabel(info.name, self)
+        self.args = [QLabel(f"{key}: {value}", self) for key, value in info.arginfo.items()]
+        self.editButton = QPushButton("EDIT")
+        self.editButton.clicked.connect(self.edit)
+        # layout
+        layout = QHBoxLayout(self)
+        self.argsLayout = QHBoxLayout()
+        self.argsLayout.addWidget(self.nameLabel)
+        for widget in self.args:
+            self.argsLayout.addWidget(widget)
+        layout.addLayout(self.argsLayout, 5)
+        layout.addWidget(self.editButton, 1)
+        self.setLayout(layout)
+
+    def updateInfo(self, info: ExperimentInfo):
+        """Updates the information by modification.
+        
+        The function edit() uses this for updating the edited arguments.
+
+        Args:
+            info: The experiment information.
+        """
+        _dismiss_items(self.argsLayout)
+        self.experimentInfo = info
+        self.nameLabel.setText(info.name)
+        for key, value in info.arginfo.items():
+            self.argsLayout.addWidget(QLabel(f"{key}: {value}", self))
+
+    def data(self) -> ExperimentInfo:
+        """Returns data to ExperimentDelegate for displaying."""
+        return self.experimentInfo
+
+    def edit(self):
+        """Shows frame for edition of experiment information.
+
+        It is called when editButton is pressed.
+        """
+        # TODO(giwon2004): Display a frame for editing the values.
+        print("Button Clicked")
+
+
+class ExperimentModel(QAbstractListModel):
+    """Model for managing the data in the submitted experiment list.
+    
+    Attributes:
+        experimentData: The list of ExperimentView widget.
+    """
+
+    def __init__(self, data: List[ExperimentInfo], parent: Optional[QWidget] = None):
+        """Extended."""
+        super().__init__(parent)
+        self.experimentData = data
+
+    def rowCount(self, parent: Optional[QModelIndex] = QModelIndex()) -> int:
```

Comment:
You can use pylint disable comment here since it overrides the parent's method.
Don't mind that the line becomes too long, because it is for technical marking using a comment.